src/cargo/sources/replaced.rs

```diff
@@ -100,11 +100,15 @@ impl<'cfg> Source for ReplacedSource<'cfg> {
     }
 
     fn describe(&self) -> String {
-        format!(
-            "{} (which is replacing {})",
-            self.inner.describe(),
-            self.to_replace
-        )
+        if self.replace_with.is_crates_io() {
+            self.inner.describe()
```

Comment:
Should we explain somewhere what's going on and why we need this special case? It's not clear why a source is replaced with crates.io and doesn't need to display the replacement to users.